.gitignore

```
@@ -84,3 +84,4 @@ android/gradlew.bat
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+/packages/Instabug-Flutter/lib/src/generated/
```

I feel like this could be more general in case we added Pigeon to other packages as well.
```suggestion
/packages/**/lib/src/generated/
```